modules/vaos/app/services/ccra/base_service.rb

```diff
@@ -23,5 +23,16 @@ def config
     def settings
       @settings ||= Settings.vaos.ccra
     end
+
+    private
+
+    ##
+    # Get appropriate headers based on whether mocks are enabled. With Betamocks we
+    # bypass the need to request tokens.
+    #
+    # @return [Hash] Headers for the request or empty hash if mocks are enabled
+    def request_headers
```

Comment:
Note: I made this consistent for both CCRA and EPS instead of copying the same code over and over.